src/acceleration/BaseQNAcceleration.cpp

```diff
@@ -216,42 +216,43 @@ void BaseQNAcceleration::updateDifferenceMatrices(
         residualMagnitude /= utils::IntraComm::l2norm(_values);
       }
 
-      PRECICE_CHECK(not math::equals(residualMagnitude, 0.0),
-                    "Attempting to add a zero vector to the quasi-Newton V matrix. This means that the residuals "
-                    "in two consecutive iterations are identical. If a relative convergence limit was selected, "
-                    "consider increasing the convergence threshold.");
+      if (math::equals(residualMagnitude, 0.0)) {
+        _forceUnderRelaxationStep = true;
+      }
```

Comment:
Let's inform the user about this. Could be an INFO or a WARNING, not sure yet.

Reply:
I guess the question is: "Is this a normal step of the procedure, or a special case we are trying to recover from and which may blow up later?".

If normal, then `INFO`. If "may blow up", then `WARNING`.

For my use case, an `INFO` would be more expected.

Reply:
With the issue Makis described, there was not even one column added to V and W, therefore a QN step can't be performed. We could add some more conditions, such as, perform a QN step if V and W already have 5 columns, otherwise do a under-relaxation step. But this becomes a weird condition imposed by the developers. I think that having very few columns might not converge quickly, but then again, would simple under-relaxation?

Reply:
Don't we anyway always do an under-relaxation for the first few iterations?

Reply:
No, only the very first iteration. But it cannot be guaranteed that if a column is not added after the first iteration, that it will be added after the second. We need to safeguard against the case where the solvers are essentially static in the beginning and V and W are not being appended with columns.